cranelift-codegen/src/binemit/stackmap.rs

```diff
@@ -52,25 +52,23 @@ impl Stackmap {
             }
         }
 
-        Stackmap::from_vec(&vec)
+        Stackmap::from_slice(&vec)
     }
 
     /// Create a vec of Bitsets from a vec of bools.
-    pub fn from_vec(vec: &Vec<bool>) -> Self {
-        let mut rem = vec.len();
-        let num_word = ((rem as f32) / 32.0).ceil() as usize;
+    pub fn from_slice(vec: &[bool]) -> Self {
+        let len = vec.len();
+        let num_word = len / 32 + (len % 32 != 0) as usize;
```

Comment:
light suggestion: can we replace all the occurrences of 32 (here and below line 64) by a const, and this const could use the size_of intrinsics on the type that's stored in the Stackmap bitset? (If required, we could alias the bitset's type with some local type, and reference it here in the size_of call)

Reply:
Done